src/store/reducers/dform/initial/styles.json

```diff
@@ -2864,7 +2864,7 @@
                     },
                     {
                       "field": "pathology.stool.chemical.ph",
-                      "label": "Ph"
+                      "label": "P.H."
```

Comment:
Is "pH" right (while my original one is also wrong)?
https://en.wikipedia.org/wiki/PH

Reply:
pH can also be used. Perhaps pH is more common so I will change to pH instead.